src/vs/base/browser/ui/selectBox/selectBox.css

```diff
@@ -6,3 +6,7 @@
 .monaco-select-box {
 	width: 100%;
 }
+
+.monaco-select-box.disabled {
```

Comment:
This should go in a .css file somewhere inside `src/sql` instead of `src/vs` in order to avoid editing VS Code's files when we don't need to